tests/test_features/test_singlefeatureviz.py

```diff
@@ -0,0 +1,188 @@
+# tests.test_features.test_singlefeatureviz
+# Test the SingleFeatureViz visualizer
+#
+# Author:   Liam Schumm <lschumm@protonmail.com>
+# Created:  Mon May 06 12:11:00 2019 -0400
+#
+# Copyright (C) 2016 District Data Labs
+# For license information, see LICENSE.txt
```

Comment:
Would you mind updating this to the new copyright line? `Copyright (C) 2019 The scikit-yb developers`